jenkins/clouds/azure/daily/tagging/tagging.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+# Read environment variables
+AZURE_CLIENT_SECRET = os.environ['AZURE_CLIENT_SECRET']
+AZURE_TENANT_ID = os.environ['AZURE_TENANT_ID']
+AZURE_CLIENT_ID = os.environ['AZURE_CLIENT_ID']
+AZURE_ACCOUNT_ID = os.environ['AZURE_ACCOUNT_ID']
```

Comment:
```suggestion
AZURE_CLIENT_SECRET = os.environ['client_secret']
AZURE_TENANT_ID = os.environ['tenant_id']
AZURE_CLIENT_ID = os.environ['client_id']
AZURE_ACCOUNT_ID = os.environ['account_id']
```

Reply:
Moved to [policy](https://github.com/redhat-performance/cloud-governance/tree/main/jenkins/clouds/azure/daily/policies) jenkins because need to run once a day because its tag all the account resource groups with global tags.